be/src/exprs/vectorized/string_functions.cpp

```diff
@@ -909,11 +909,11 @@ static inline ColumnPtr repeat_const_not_null(const Columns& columns, const Bina
     } else {
         raw::make_room(&dst_offsets, num_rows + 1);
         dst_offsets[0] = 0;
-        size_t reserved = times * src_offsets.back();
+        size_t reserved = static_cast<size_t>(times) * src_offsets.back();
         if (reserved > OLAP_STRING_MAX_LENGTH * num_rows) {
```

Comment:
here, operations may overflow, i.e. OLAP_STRING_MAX_LENGTH(1024*1024)*16384.

Reply:
@satanson The type of `num_rows` is `size_t`, so `OLAP_STRING_MAX_LENGTH * num_rows` cannot be overflow.